.changeset/olive-rabbits-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro-integration-kit": patch
```

Comment:
```suggestion
"astro-integration-kit": minor
```

Reply:
I was a bit on the fence about this one because it doesn't break anything. But let's go